src/features/collapsible-sidebar/CollapsibleSidebar.js

```diff
@@ -36,22 +37,66 @@ type Props = {
     htmlAttributes?: Object,
 };
 
-const CollapsibleSidebar = (props: Props = { expanded: false }) => {
-    const { children, className, expanded, htmlAttributes } = props;
-
-    const classes = classNames(
-        {
-            'is-expanded': expanded,
-        },
-        'bdl-CollapsibleSidebar',
-        className,
-    );
-
-    return (
-        <aside className="bdl-CollapsibleSidebar-wrapper" {...htmlAttributes}>
-            <StyledNav className={classes}>{children}</StyledNav>
-        </aside>
-    );
-};
+class CollapsibleSidebar extends React.Component<Props> {
+    navRef: { current: null | HTMLElement } = React.createRef();
+
+    focusNextEl = () => {
```

Comment:
Maybe nit, but this is nearly identical to `focusPreviousEl()`. Any way this could take a parameter like `direction` or similar?